src/containers/TemplateBuilder/shared/OperationContextHelpers.tsx

```diff
@@ -187,22 +187,29 @@ export const createApplication: CreateApplicationHelper =
 export const exportTemplate: TemplateOperationHelper = async (
   { id, snapshotName },
   setErrorAndLoadingState
-) =>
-  await safeFetch(
+) => {
+  const result = await safeFetch(
     getServerUrl('snapshot', {
       action: 'take',
       name: snapshotName,
       options: templateExportOptionName,
     }),
-    getFitlerBody(id),
+    getFilterBody(id),
     setErrorAndLoadingState
   )
 
+  // Delete the snapshot cos we don't want snapshots page cluttered with individual templates
+  if (result)
+    safeFetch(getServerUrl('snapshot', { action: 'delete', name: snapshotName }), {}, () => {})
+
+  return result
+}
+
```

Comment:
Unrelated -- just a typo and small change as per the comment (to match what it already does on import)